main.js

```diff
@@ -0,0 +1,67 @@
+var HanoiTower = function(disk, start, end, via) {
+var board = [[3, 2, 1], [], []];
+
+  var displayDisc = board.map(function (row) {
+        console.log(row.join(''));
+  });
+
+  var moveDisc = board.filter(function (start, end) {
+    var disk = board[0].pop(2);
```

Comment:
the `Array.pop()` method doesn't require any arguments, it just returns the last item in an array. Just a heads up